### PR TITLE
Update to raft v1.3.10

### PR DESCRIPTION
**Pull request by nisainan**

### CODE REVIEW

File: transport.go [hunk 1]
```diff
@@ -3,45 +3,140 @@ package libp2praft
...
-	case strings.Contains(t, "[WARN]"):
-		logger.Warn(strings.TrimPrefix(t, "[WARN]  raft-net: "))
-	case strings.Contains(t, "[ERR]"):
-		logger.Error(strings.TrimPrefix(t, "[ERR] raft-net: "))
-	case strings.Contains(t, "[INFO]"):
-		logger.Info(strings.TrimPrefix(t, "[INFO] raft-net: "))
+var raftLogger = logging.Logger("raftlib")
+
+// this implements github.com/hashicorp/go-hclog
+type hcLogToLogger struct {
```
Comment (hsanjuan) [inline transport.go:26]: Can you please export this type? (It will require to add comments to the exported methods for checks to pass).

**Review by hsanjuan — CHANGES_REQUESTED** [review]

File: transport.go (hunk 1 above)
Comment (nisainan) [reply transport.go]: updated, have a try plz

**Review by hsanjuan — APPROVED** [review]
Thanks. Before I tag a new release I will test that this is not causing any breakage with any Raft state using the previous version, but I think it is all good.

**Comment by hsanjuan** [comment]
Thanks so much @nisainan !

**Pull request merged**